PySDM_examples/Lowe_et_al_2019/aerosol.py

```diff
@@ -0,0 +1,162 @@
+        molar_volumes = {i: molar_masses[i] / densities[i] for i in compounds}
+
+        _masked = {k: (not is_organic[k]) * volfrac[k] for k in compounds}
+        volume_fractions_of_just_inorg = {k:_masked[k] / np.sum(list(_masked.values())) for k in compounds}
+
+        if model == 'film':
+            ns_per_vol = (1 - f_org_volume(mass_fractions))  * np.sum(
+                ionic_dissociation_phi[i] * volume_fractions_of_just_inorg[i] / molar_volumes[i] for i in compounds
+            )
+        elif model == 'bulk':
+            ns_per_vol = np.sum(ionic_dissociation_phi[i] * volfrac[i] / molar_volumes[i] for i in compounds)
+        else:
+            raise AssertionError()
+        kappa[model] = ns_per_vol * const.Mv / const.rho_w
+
+    return kappa
+
+class _Aerosol:
+    pass
+
+
+@strict
+class AerosolMarine(_Aerosol):
+    Aitken = {'palmitic': .2, 'SOA1': 0, 'SOA2': 0, '(NH4)2SO4': .8, 'NH4NO3': 0, 'NaCl': 0}
+    Accumulation = {'palmitic': .2, 'SOA1': 0, 'SOA2': 0, '(NH4)2SO4': 0, 'NH4NO3': .8, 'NaCl': 0}
+    def __init__(self):
+        self.aerosol_modes_per_cc = (
+        {
+            'f_org': f_org_volume(self.Aitken),
+            'kappa': kappa(self.Aitken),
+            'spectrum': spectra.Lognormal(
+                norm_factor=226 / si.cm ** 3,
+                m_mode=19.6 * si.nm,
+                s_geom=1.71
+            )
+        },
+        {
+            'f_org': f_org_volume(self.Accumulation),
+            'kappa': kappa(self.Accumulation),
+            'spectrum': spectra.Lognormal(
+                norm_factor=134 / si.cm ** 3,
+                m_mode=69.5 * si.nm,
+                s_geom=1.7
+            ),
+        }
+    )
+    color = 'dodgerblue'
+
+@strict
+class AerosolBoreal(_Aerosol):
+    # note: SOA1 or SOA2 unclear from the paper
+    Aitken = {'palmitic': 0, 'SOA1': 0.668, 'SOA2': 0, '(NH4)2SO4': 0.166, 'NH4NO3': 0.166, 'NaCl': 0}
+    Accumulation = {'palmitic': 0, 'SOA1': 0, 'SOA2': 0.668, '(NH4)2SO4': 0.166, 'NH4NO3': 0.166, 'NaCl': 0}
+    def __init__(self):
+        self.aerosol_modes_per_cc = (
+        {
+            'f_org': f_org_volume(self.Aitken),
+            'kappa': kappa(self.Aitken),
+            'spectrum': spectra.Lognormal(
+                norm_factor=1100 / si.cm ** 3,
+                m_mode=22.7 * si.nm,
+                s_geom=1.75
+            )
+        },
+        {
+            'f_org': f_org_volume(self.Accumulation),
+            'kappa': kappa(self.Accumulation),
+            'spectrum': spectra.Lognormal(
+                norm_factor=540 / si.cm ** 3,
+                m_mode=82.2 * si.nm,
+                s_geom=1.62
+            )
+        },
+    )
+    color = 'yellowgreen'
+
+
+@strict
+class AerosolNascent(_Aerosol):
+    Ultrafine = {'palmitic': 0, 'SOA1': .52, 'SOA2': 0, '(NH4)2SO4': .48, 'NH4NO3': 0, 'NaCl': 0}
+    Accumulation = {'palmitic': 0, 'SOA1': 0, 'SOA2': .3, '(NH4)2SO4': .6, 'NH4NO3': 0, 'NaCl': 0}  # TODO #586: just approx from plot
```

Comment:
```suggestion
    Accumulation = {'palmitic': 0, 'SOA1': 0, 'SOA2': .3, '(NH4)2SO4': .7, 'NH4NO3': 0, 'NaCl': 0}
```

30% SOA2 and 70% ammonium sulfate -- source https://github.com/SamJLowe/NatComms_OrgSurfaceClouds/blob/4a9d481d07d618321164781a4da042fc97cde719/Fig1/FIG1_SIZEDIST_KOHLER.m#L53-L59